drivers/mfd/mfd_npm1300.c

```diff
@@ -175,7 +176,12 @@ static int mfd_npm1300_init(const struct device *dev)
 		return ret;
 	}
 
-	return mfd_npm1300_reg_write(dev, SHIP_BASE, SHIP_OFFSET_LPCONFIG, config->lp_reset);
+	ret = mfd_npm1300_reg_write(dev, SHIP_BASE, SHIP_OFFSET_LPCONFIG, config->lp_reset);
+	if (ret < 0) {
+		return ret;
+	}
+
+	return mfd_npm1300_reg_write(dev, SHIP_BASE, SHIP_OFFSET_CFGSTROBE, 1U);
```

Comment:
btw, not asking you to change it, but I normally like explicit `return 0` on anything other than wrappers to avoid diffs like this, I'd still do

```c
ret = mfd_npm1300_reg_write(dev, SHIP_BASE, SHIP_OFFSET_CFGSTROBE, 1U);
if (ret < 0) {
        return ret;
}

return 0;
}
```

here

Reply:
I like that, makes it easier to grok later changes, especially for big init functions.